conans/cli/commands/create.py

```diff
@@ -0,0 +1,168 @@
+import os
+
+from conans.cli.command import conan_command, COMMAND_GROUPS, OnceArgument
+from conans.cli.commands import make_abs_path
+from conans.cli.commands.export import common_args_export
+from conans.cli.commands.install import _get_conanfile_path
+from conans.cli.common import get_lockfile, get_profiles_from_args, _add_common_install_arguments, \
+    _help_build_policies
+from conans.cli.conan_app import ConanApp
+from conans.cli.formatters.graph import print_graph_basic, print_graph_packages
+from conans.cli.output import ConanOutput
+from conans.client.conanfile.build import run_build_method
+from conans.client.graph.printer import print_graph
+from conans.errors import ConanException, conanfile_exception_formatter
+from conans.util.files import chdir, mkdir
+
+
+@conan_command(group=COMMAND_GROUPS['creator'])
+def create(conan_api, parser, *args):
+    """
```

Comment:
@lasote this is the main method to read. This PR is basically moving the legacy ``create`` here, using the new PythonAPI, and executing things a bit more "flat". This is a large command method, but I prefer it this way at the moment, being explicit.